pkg/storaged/overview.js

```diff
@@ -417,6 +418,21 @@ define([
                                 }
                               }
                           ],
+                          ChangeHandler: function (dialog) {
+                              // hide the chunk size selector for RAID 1, as the chunk size setting doesn't make sense
```

Comment:
What about using the 'visible' property in the 'chunk' field definition?  That should be able to cleanly handle showing/hiding of the control.

Instead of forcing the 'chunk' field to 0, I would handle this case in the 'action' function.  I don't think we want to have "0" option in the dropdown, or do we?  Does it make sense to pass 0 for RAID5, for example?

Untested:
```